doc/README

```diff
@@ -0,0 +1,14 @@
+===========================================
+NISTATS: functional MRI analysis in python
+===========================================
+
+This repository contains code and documentation to perform fMRI analysis. The requirements are ...
```

Comment:
I would say more mid February? that would be really nice to also tackled pending small issues.